src/index.ts

```diff
@@ -10,11 +10,11 @@ export function connect(
   options?: SocketOptions,
 ): Socket {
   if (typeof address === 'string') {
-    const url = new URL(address);
+    const url = new URL("https://" + address);
```

Comment:
I think we need a little more here. What if the `address` is something like `tcp://0.0.0.0:00`? We wouldn't want that prepended by `https`

Reply:
This is something we should clarify in the spec: protocol shouldn't be supported in the `address`. It should always be "hostname:port".

Reply:
Tracking here https://github.com/wintercg/proposal-sockets-api/issues/13